### ISSUE TITLE
No defaults when removing links section

### ISSUE THREAD

**Issue by jamminuesa** [bastienwirtz/homer#575]
Hi there, 
I believe there is a bug when the links section is set to false. In this scenario, the defaults section for layout and colourTheme is ignored. 
This is my config.yml file

`title: "Dashboard"
subtitle: "Homer"
logo: "assets/icons/pwa-512x512.png"

header: true
footer: false

defaults:
  layout: list # Either 'columns', or 'list'
  colorTheme: dark # One of 'auto', 'light', or 'dark'
  
theme: default

links: false`
Output: Theme light and columns layout

If I change links: false to links: [], then the output is theme dark and list layout

**Comment by broizter** [comment]
Yep same issue here. My workaround is to assign the same color values to both the light and dark theme in the config since I only ever wanna use the dark theme anyway.

**Comment by stefanes** [comment]
Can confim, no links section/navbar - no defaults... (tested in private windows across various browsers)

**Comment by go2tom42** [comment]
confirmed
